### ISSUE TITLE
Replace issue_template.md with multiple alternate templates

### ISSUE THREAD

**Issue by cmungall** [INCATools/ontology-development-kit#162]
Example of how to do this
https://github.com/ewels/MultiQC/tree/master/.github

Thanks @olgabot for the suggestion!

**Comment by drseb** [comment]
Some suggestions from my HPO perspective:

- Adding term
- Adding synonym
- Subclass revision
- Logical definition revision
- Textual definition revision
- Obsoletion

**Comment by pnrobinson** [comment]
Also: new annotation+new term (i.e., not only add a new term, but suggest what the domain objects are that should be annotated to it).

**Comment by drseb** [comment]
Correct. But maybe (to make things simpler) this should simply be added to "Adding term".

**Comment by matentzn** [comment]
As the trackers are used for ontology pipeline maintenance, I would like to add:
* Questions about how to use the ontology
* Feature requests for release pipeline (adding new QC test, new release artefact)
* Bug report for ontology, release and continuous integration pipeline ("ontology is missing obo tag x! WHY?", "the pipeline generates labels twice", etc. 

It would be especially great if issues thus created could be tagged accordingly and, even better, assigned immediately to some configurable (set of) person(s).

**Comment by cmungall** [comment]
Great suggestions! I'm not sure if this feature allows auto-assigning people (but it probably wouldn't be hard to get a bot to do it).

I wonder if we should provide all choices or keep to a more manageable set - and also focused around where we can provide simple ready to fill in fields. I can see how to do that for New-X or Modified-X, but less so for pipeline issues.

**Comment by drseb** [comment]
+1 for a manageable set

**Comment by drseb** [comment]
> I'm not sure if this feature allows auto-assigning people

couldn't we just mention the right people/teams via the @ - notation?

**Comment by matentzn** [comment]
I am happy for manageable :P It might have been a bit of an egoistic suggestion, because I keep having to delete the term request blurp on all the ontologies I interact with. But maybe that is not so bad. Keep it short! :)

**Comment by nlharris** [comment]
> * Adding term
> * Adding synonym
> * Subclass revision
> * Logical definition revision
> * Textual definition revision
> * Obsoletion

Before these get made into the issue types, I suggest using consistent phrasing:
* Add term
* Add synonym
* Revise subclass
* Revise logical definition
* Revise textual definition
* Obsolete a term

**Comment by drseb** [comment]
@pnrobinson @cmungall we have now implemented this in HPO. thanks @nlharris for the suggestions

**Comment by drseb** [comment]
see https://github.com/obophenotype/human-phenotype-ontology/issues/new/choose

**Comment by pnrobinson** [comment]
very nice! :-0

**Comment by pbuttigieg** [comment]
Hi all, good stuff. I'll be implementing these in ENVO.

I'm with @nlharris on consisting phrasing and prefer the imperative tone: 

>Add term
Add synonym
Revise subclass
Revise logical definition
Revise textual definition
Obsolete a term

**Comment by cmungall** [comment]
nice!

**Comment by drseb** [comment]
Hi all, 

just wanted to let you know that it gets even cooler. You can automatically assign people and automatically assign labels in the templates. See [here](https://github.com/obophenotype/human-phenotype-ontology/tree/master/.github/ISSUE_TEMPLATE) how to do it and click somewhere [here](https://github.com/obophenotype/human-phenotype-ontology/issues/new/choose) to see it in action.

Best,
Seb

**Comment by cmungall** [comment]
ooooh, awesome!!

**Comment by cmungall** [comment]
btw: this is an excellent opportunity to standardize on the use of github labels across repos.

For example, if we standardize on "obsolete" then we can do a cross-repo search:

https://github.com/search?q=label%3A%22obsolete%22

(so far this just returns GO, but @drseb has introduced this for HPO)

**Comment by pbuttigieg** [comment]
> btw: this is an excellent opportunity to standardize on the use of github labels across repos.

+1
